.github/workflows/deploy-to-control-plane-review-app.yml

```
@@ -70,7 +71,28 @@ jobs:
           fetch-depth: 0
 
       - name: Validate Required Secrets and Variables
-        uses: ./.github/actions/validate-required-vars
+        shell: bash
+        run: |
+          missing=()
+          
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)